boot/export_test.go

```diff
@@ -134,6 +134,9 @@ var (
 	PredictableBootChainsEqualForReseal = predictableBootChainsEqualForReseal
 	BootAssetsToLoadChains              = bootAssetsToLoadChains
 	BootAssetLess                       = bootAssetLess
+	BootChainsToFile                    = writeBootChains
```

Comment:
the two sides don't match anymore

Reply:
One of these days I'm going to improve godoctor to catch those.